src/index.js

```diff
@@ -1,3 +1,148 @@
-// eslint-disable-next-line no-unused-vars
-import _ from 'lodash';
 import './style.css';
+
+let gameId = ''; // Store the ID of the game created by you.
+
+// Function to display an error message in the DOM.
+const displayErrorMessage = (message) => {
+  const errorContainer = document.querySelector('.error-container');
+  errorContainer.textContent = message;
+};
+
+// Function to create a new game and save gameId to local storage.
+const createGame = async () => {
+  try {
+    const response = await fetch(
+      'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/',
+      {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({ name: 'Barcelona' }),
+      },
+    );
+    const data = await response.json();
+    const gameId = data.result.split(':')[1].trim();
+
+    // Save gameId to local storage.
+    localStorage.setItem('gameId', gameId);
+  } catch (error) {
+    displayErrorMessage(`Error creating game: ${error.message}`);
+  }
+};
+
+// Function to load gameId from local storage on page load.
+const loadGameIdFromLocalStorage = () => {
+  const storedGameId = localStorage.getItem('gameId');
+  if (storedGameId) {
+    gameId = parseInt(storedGameId, 10); // Parse the stored value as an integer
+  }
```

Comment:
- [ ] You're on the right track with adding new scores upon clicking the submit button. However, there's a small issue to address. Currently, the scores are not being added to the actual game ID that was created by the `creategame()` function. Instead, they are being associated with a game ID of `NaN`. To clarify, this happens because the game ID contains non-numeric characters, causing `parseInt()` to fail and return `NaN`. Consequently, your submit action is linked to the `NaN` game ID instead of the intended one. 
![image](https://github.com/efmomoh/Leaderboard/assets/55840999/9b3df5fa-d48b-429c-ac77-c4e41b78c7a8)

To resolve this issue, you can consider using the actual game ID without converting to a number. By making this adjustment, you'll ensure that the scores are added to the correct game ID, providing accurate and organized data within your project. Keep up the great work, and thank you for addressing this issue! 🌟🎮